CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Version 0.3.2 (2024-03-21)
+
+This is a minor bugfix release which fixes loading install receipts which contain UTF-8 byte order marks.
```

Comment:
This was the release note for 0.3.1; can leave it out in favour of the notes below.